README.md

```diff
@@ -18,6 +18,29 @@ npm i --save @uniswap/analytics
 
 Make sure the environment variable `REACT_APP_STAGING` is set to `false` in production.
 
+## Development
+
+When making changes to this package, first ensure your changes are designed as needed by installing your changes from a tarball. To generate the tarball (with version `0.0.1` to indicate a test version) use the following command:
+
+```bash
+yarn tarball
+```
+
+To install in the relevant repo, copy the tarball to the root of your project's `package.json` and run the following commands in your project's root:
+
+```bash
+# yarn
+yarn cache clean
+yarn add file:uniswap-analytics-dev.tgz
+
+# npm
+npm install uniswap-analytics-dev.tgz
+```
+
+Alternatively, you can install with a direct file reference from your local `analytics` repository.
```

Comment:
We could add a sample command here as stated above `yarn add file:<file_path>/@uniswap/analytics`

Reply:
That latter part of the path even would vary based on your clone path, so I think it's not that useful. If there's any confusion we can add clarification, but development should be minimal on the infra side!